site/app/Resources/views/emailer.html.twig

```diff
@@ -0,0 +1,154 @@
+                                                <tr>
+                                                    <td align="center" style="font-size: 25px; font-family: Helvetica, Arial, sans-serif; color: #333333;" class="padding-copy">Welcome to Librecores</td>
+                                                </tr>
+                                                <tr>
+                                                    <td align="center" style="padding: 20px 0 0 0; font-size: 16px; line-height: 25px; font-family: Helvetica, Arial, sans-serif; color: #666666;" class="padding-copy">LibreCores is your gateway to free and open source digital designs and other components that you can use and re-use in your digital designs</td>
+                                                </tr>
+                                            </table>
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td>
+                                            <!-- BULLETPROOF BUTTON -->
+                                            <table width="100%" border="0" cellspacing="0" cellpadding="0" class="mobile-button-container">
+                                                <tr>
+                                                    <td align="center" style="padding: 25px 0 0 0;" class="padding-copy">
+                                                        <table border="0" cellspacing="0" cellpadding="0" class="responsive-table">
+                                                            <tr>
+                                                                {% autoescape false %}
+                                                                    <td align="center"><a href="{{ confirmationUrl }}" target="_blank" style="font-size: 16px; font-family: Helvetica, Arial, sans-serif; font-weight: normal; color: #ffffff; text-decoration: none; background-color: #48CFAD; border-top: 15px solid #48CFAD; border-bottom: 15px solid #48CFAD; border-left: 25px solid #48CFAD; border-right: 25px solid #48CFAD; border-radius: 3px; -webkit-border-radius: 3px; -moz-border-radius: 3px; display: inline-block;" class="mobile-button">Confirm Your Email &rarr;</a></td>
+                                                                {% endautoescape %}
+                                                            </tr>
+                                                        </table>
+                                                    </td>
+                                                </tr>
+                                            </table>
+                                        </td>
+                                    </tr>
+                                </table>
+                            </td>
+                        </tr>
+                    </table>
+                </td>
+            </tr>
+        </table>
+
```

Comment:
It's better (and required in some legislations) to add a mail footer: https://codepen.io/rodriguezcommaj/pen/RNPzwr

Please add LibreCores contact address and why they received this email something like "you have received this email as you or someone posing as you have signed-up at LibreCores"

Reply:
I want the content for the mail footer. i have checked the librecores website but didn't get any address which i can use in the footer